app/Domain/Goalcanvas/Templates/canvasComment.blade.php

```diff
@@ -0,0 +1,74 @@
+@extends($layout)
+@section('content')
+
+@php
+
+/**
+* canvasComment.inc template - Generic template for comments
+*
+* Required variables:
+* - $canvasName Name of current canvas
+*/
+
+
+$canvasName = "goal";
+
+$id = "";
+if (isset($canvasItem['id']) && $canvasItem['id'] != '') {
+$id = $canvasItem['id'];
+}
+@endphp
+
+<script type="text/javascript">
+    window.onload = function() {
+        if (!window.jQuery) {
+            //It's not a modal
+            location.href = "<?= BASE_URL ?>/<?= $canvasName ?>canvas/showCanvas?showModal=<?php echo $canvasItem['id']; ?>";
+        }
+    }
+</script>
+
+<div class="showDialogOnLoad" style="display:none;">
+
+    <h4 class="widgettitle title-light" style="padding-bottom: 0">
+        <i class="fas {{ $canvasTypes[$canvasItem['box']]['icon'] }}"></i>
+        {{ $canvasTypes[$canvasItem['box']]['title'] }}
+    </h4>
+    <hr style="margin-top: 5px; margin-bottom: 15px;">
+
+    {!! $tpl->displayNotification() !!}
+
+    <h5 style="padding-left: 40px"><strong>{{ $tpl->e($canvasItem['description']) }}</strong></h5>
```

Comment:
`$tpl->e()` is not required when using {{ }} So you can just do `{{ $canvasItem['description'] }}